### ISSUE TITLE
How to generate  test wav?

### ISSUE THREAD

**Issue by imucsliurui** [ibab/tensorflow-wavenet#206]
I run  "python generate.py --wav_out_path=generate.wav --sample 16000 model.ckpt-12850"  and get the wav file .
But i don't know where is the test data corresponding the generated wav file?
After the completion of network training, how to choose my own test set to generate test wav file?

Thank you everyone!

**Comment by akademi4eg** [comment]
generate.py starts with random noise and generates some random audio. Usually that audio is assessed subjectively by listening. If you want to start from initial samples of some file, you can use --wav_seed=some_sound.wav and --window=number_of_samples parameters. It will use number_of_samples samples from provided wav-file and then generate from them. Note that after initial 100-200ms, depending on your receptive field length, the audio will deviate more and more from your wav-file.

If you want to test the model on some separate test set, you can write your script and use WaveNetModel.loss method. It will calculate deviation for each sample properly.

**Comment by imucsliurui** [comment]
@akademi4eg OK! Thank you very much！

**Comment by imucsliurui** [comment]
@akademi4eg Ok！ Thank you very much！
I don't quite understand your answer because of personal problems， your answer can  achieved "local conditioning"?  I would like to ask you how to achieve local conditional for wavenet.
Given  the additional input： for example test fullcontext lab file, How to generate the corresponding voice？
Thank you！

**Comment by akademi4eg** [comment]
Current version has 2 modes: unconditioned and global conditioning (e.g. speaker voice). Local conditioning (e.g. text) is not implemented yet.

**Comment by imucsliurui** [comment]
Ok!Thank you very much!
